eng/pipelines/coreclr/templates/helix-queues-setup.yml

```diff
@@ -26,18 +26,22 @@ jobs:
     runtimeFlavorDisplayName: ${{ parameters.runtimeFlavorDisplayName }}
     helixQueues:
 
+    # iOS/tvOS simulator x64/x86
+    - ${{ if in(parameters.platform, 'iOSSimulator_x64', 'iOSSimulator_x86', 'tvOSSimulator_x64') }}:
+      - OSX.1015.Amd64.Open
```

Comment:
for my knowledge, will this be used for x86 too ?

Reply:
There are no plans, as far as I know, for an iOS Simulator x86 lane. This was duplicated from `pipelines/libraries/helix-queues-setup.yml`. I can remove the definitions platforms we definitely are not testing as part of https://github.com/dotnet/runtime/issues/33381.